File: tests/go/cases/case_172_deferred_sink_gate_08.go
package escape_tests

var retainedCase172 = []map[string]string{}

func Case172DeferredSinkGate08(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "deferred_sink_gate_08",
		"entity": "stress",
		"stage": "evaluation",
		"input": raw,
		"checksum": "deferred_sink_gate_08:" + raw,
	}
	if len(raw) > 0 && raw[0] == 'x' {
		// ESCAPE: conditional path retention.
		retainedCase172 = append(retainedCase172, payload)
	}
	return "ok"
}

File: tests/python/cases/case_172_deferred_sink_gate_08.py
"""
Task case 172: deferred_sink_gate_08 false-positive/false-negative stress pattern.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_172_deferred_sink_gate_08(input_data):
    task_name = 'deferred_sink_gate_08'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'stress',
        'stage': 'evaluation',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    if raw.startswith('x'):
        # ESCAPE: conditional sink persists payload only on specific branch.
        RETAINED_AUDIT.append(payload)
    return 'ok'
